qubesmanager/tests/instructions

```diff
@@ -0,0 +1,5 @@
+To run qube manager and backup tests:
+python3 test_name.py -v
+
+To run global settings tests:
+sudo systemctl stop qubesd; sudo -E python3 test_global_settings.py -v ; sudo systemctl start qubesd
```

Comment:
Better move it to the main directory, maybe even into `README.md`. Otherwise probably no one will even know that there are some tests...